recipes/ttfunk/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "ttfunk" %}
+{% set version = "1.7.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+    sha256: 2370ba484b1891c70bdcafd3448cfd82a32dd794802d81d720a64c15d3ef2a96
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+
+requirements:
+  host:
+    - ruby
+  run:
+    - {{ pin_compatible('ruby', max_pin='x.x') }}
+
+test:
+  commands:
+    - ruby -r {{ name }} -e 'exit 0'
+
+about:
+  home: https://prawnpdf.org/
+  license: GPL-2.0-or-later
+  license_family: GPL
+  license_file:
+    - LICENSE
+    - GPLv2
+    - GPLv3
+  summary: |
+    Font Metrics Parser for the Prawn PDF generator
+  description:
+  doc_url: https://prawnpdf.org/docs/ttfunk/{{ version }}
```

This is a valid URL for the latest doc, the URL with jinja isn't.

```suggestion
  doc_url: https://prawnpdf.org/docs/ttfunk/
```